sass/blocks/_blocks.scss

```diff
@@ -135,6 +135,7 @@
 			outline: none;
 			color: white;
 			outline: none;
+			text-decoration: none;
```

Comment:
This button underline was actually taken care of separately via #363 earlier today (Sorry about that — I hadn't realized your PR was tackling all buttons globally). This will add a duplicate rule to the file, so it would need to be removed.